src/input/internal.tsx

```diff
@@ -101,7 +101,7 @@ function InternalInput(
 
   const attributes: React.InputHTMLAttributes<HTMLInputElement> = {
     'aria-label': ariaLabel,
-    'aria-labelledby': ariaLabelledby,
+    'aria-labelledby': ariaLabel && !rest.ariaLabelledby ? undefined : ariaLabelledby,
```

Comment:
I would add a code comment to explain the expectations. Alternatively, a util might make sense as we have integration with the form field in several components

Reply:
I'll add a comment for now. We have a different treatment of this property across components, so we need to agree on a universal solution. But the issue with input is blocking customers